src/modules/kvs-watch/kvs-watch.c

```diff
@@ -172,8 +172,12 @@ static void namespace_destroy (void **data)
         int saved_errno = errno;
         commit_destroy (nsm->commit);
         zlistx_destroy (&nsm->watchers);
-        if (nsm->subscribed)
-            (void)flux_event_unsubscribe (nsm->ctx->h, nsm->topic);
+        if (nsm->subscribed) {
+            flux_future_t *f;
+            if (!(f = flux_event_unsubscribe_ex (nsm->ctx->h, nsm->topic, 0)))
+                flux_log_error (nsm->ctx->h, "flux_event_unsubscribe_ex");
```

Comment:
Use flags=FLUX_RPC_NORESPONSE to avoid having the broker generate a response you don't intend to look at.